src/swish/db.ss

```diff
@@ -484,11 +488,9 @@
      (match (osi_prepare_statement* (database-handle db) sql
```

Comment:
Looks like line 480 should be storing `handle` now instead of 0 where we restore `db` on failure to close.

Reply:
Nice find, thanks!